ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/transform/CompressPayload.java

```diff
@@ -0,0 +1,44 @@
+package com.mozilla.telemetry.ingestion.sink.transform;
+
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.PubsubMessage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.function.Function;
+import java.util.zip.GZIPOutputStream;
+
+public enum CompressPayload implements Function<PubsubMessage, PubsubMessage> {
+  gzip {
```

Comment:
Do we have other enums in this codebase yet? I'm feeling some regret about the decision to use lowercase enum values in ingestion-beam. It looks nicer on the command-line, but breaks with the style used by enum options used by Beam and breaks with established Java convention for all-caps enum values.

Reply:
we have `PubsubMessageToObjectNode.Format`, which I'm planning to refactor as part of implementing `OUTPUT_FORMAT=payload`. I'm fine with switching to all-caps.

Reply:
I switched both enums to use all-caps values

Reply:
(and use `.toUpperCase()`, for backward compatiblity)